pkg/asset/installconfig/platform.go

```diff
@@ -62,10 +85,21 @@ func (a *Platform) Name() string {
 }
 
 func (a *Platform) queryUserForPlatform() (string, error) {
+	sort.Strings(validPlatforms)
 	prompt := asset.UserProvided{
-		Prompt: &survey.Select{
-			Message: "Platform",
-			Options: validPlatforms,
+		Question: &survey.Question{
+			Prompt: &survey.Select{
+				Message: "Platform",
+				Options: validPlatforms,
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
```

Comment:
> Does this validation ever actually catch an error?

Yes, because we use it on the environment-variable input too, when users specify the value that way.

Reply:
Ahhhhhhhhhhh, yes.